src/live/test-runner.js

```diff
@@ -89,12 +89,23 @@ class LiveModeRunner extends Runner {
             .then(() => this.runTests(true));
 
 
-        return this._waitInfinite().then(() => {
-            this.preventRunCall = false;
-        });
+        return this._waitInfinite()
```

Comment:
`_waitInfinite` is bad name. When the promise of this method should be resolved? After infinite?

I propose to use `_waitUntilExit` name or something else.

Reply:
Yes. Let's do this later.